crates/sui-analytics-indexer/Cargo.toml

```diff
@@ -0,0 +1,68 @@
+[package]
+name = "sui-analytics-indexer"
+version.workspace = true
+authors = ["Mysten Labs <build@mystenlabs.com>"]
+license = "Apache-2.0"
+publish = false
+edition = "2021"
+
+
+[dependencies]
+anyhow.workspace = true
+async-trait.workspace = true
+axum.workspace = true
+backoff.workspace = true
+bcs.workspace = true
+chrono.workspace = true
+clap.workspace = true
+csv.workspace = true
+tap.workspace = true
+diesel.workspace = true
```

Comment:
nit: diesel seems not used?

Reply:
yeah thanks, will remove and probably more. I was looking for a tool that could tell unused dependencies but could not find any. If you know something let me know please